Userland/Services/WebServer/main.cpp

```diff
@@ -92,7 +92,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     TRY(server->listen(ipv4_address.value(), port));
 
-    outln("Listening on {}:{}", ipv4_address.value(), port);
+    out("Listening on ");
+
+    auto url = URL(String::formatted("http://{}:{}", ipv4_address.value(), port));
```

Comment:
When would a serialized IP+port URL ever be different from the string-formatted value?

Reply:
I don't understand what you mean by that. Are you suggesting to use some other api to create the url?

Reply:
In other words: Why do we need to create a URL object from a string only to immediately serialize it back to a string that I don't expect to ever be different from the input string?

Reply:
oh duh, I was thinking like 10 steps ahead to where I'm handing a url object in LibVT again